cross_list_courses/templates/cross_list_courses/partials/list.html

```diff
@@ -0,0 +1,30 @@
+{% verbatim %}
+
+<nav>
+    {% endverbatim %}
+    <h3><a href="{% url 'dashboard_account' %}">Admin Console</a>
+        <small><i class="fa fa-chevron-right"></i></small> Cross Listing Single Pair of Courses</h3>
+    {% verbatim %}
+</nav>
+
+<main class="float-left col-60">
+  <div class="container-fluid">
```

Comment:
I don't think we should have multiple `container` classes in a single HTML document. I think we're doing this elsewhere but it is not a practice bootstrap 'supports': http://getbootstrap.com/css/#overview-container
> Note that, due to padding and more, neither container is nestable.